compute/rn/release-information/release-notes-22-06-update1.adoc

```diff
@@ -0,0 +1,64 @@
+== 22.06 Update 1 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.197
+
+|Codename
+|Kepler, 22.06 Update 1
+//Tentative date
+|Release date
+|Jul 27, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|5aa618314e176d03e559e58d2eba50959365cdc145cba99f5d47d90737d233bf
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, Fixes, and Performance Enhancements
+//GithubIssue PCSUP-issue
+// GH#39791 #PCSUP-10025 On-prem
+* Fixed the broken pipe error that occured while https://prisma.pan.dev/api/cloud/cwpp/images#operation/get-images-download[downloading a large image CSV] for secondary consoles when using Projects. The error was fixed by extending the HTTP client timeout value.
+// GH#40032 #PCSUP-10097 On-prem only
+* Fixed the welcome tour screen for new users who don't have an administrator role.
+// GH#39668 #PCSUP-9482
+* Fixed an issue wherein the Defenders blocked application deployments on SELinux due to incorrect SELinux labeling on proxy `runc`. The issue was fixed by applying the original runc's SELinux label to the created `runc` proxy binary.
+// GH#39821 On-prem only
+* Fixed the validity period error of https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/configure/certificates[self-signed certificates]. The limit of 365 has been waved off and the value can now be a whole number greater than or equal to 1.
+// GH#39434 PCSUP-9587 + 1749
+* Fixed an error in evaluating the custom compliance checks for registry with CRI-O (Dockerless) Defender. Custom checks are now run on the materialized image path.
+// GH#39351 PCSUP-9555
+* Fixed error that overwrote the communication port after upgrading a Defender with a custom port from the Prisma Cloud Console UI.
+// GH#37579 PCSUP-8519
+* Fixed an issue that showed different fixes for the same CVE on a single image. Each CVE vulnerability is consolidated and grouped according to OS version for each image and package.
+// GH#38819 PCSUP-9069
+* Fixed the filter to list all the tags when multiple images have the same digest.
+//GH#39581
+* Added support for GKE version 1.24.1 running on ARM64 architecture.
```

Comment:
@Pubs-MV I suggest emphasizing the new features and supported orchestrators on top of the bug fixes, maybe by moving them to the head of the list or listing them under another topic. 
Specifically, regarding the GKE ARM support, we can also add a link to the [Blogpost](https://www.paloaltonetworks.com/blog/prisma-cloud/supports-arm-workloads-on-google-cloud-and-gke/#:~:text=Prisma%20Cloud%20Supports%20Arm%20Workloads%20on%20Google%20Cloud%20and%20GKE,-Cloud%20Workload%20Protection&text=Prisma%20Cloud%20integrates%20with%20GKE,Google%20Kubernetes%20Engine%20(GKE).)

Reply:
yes